dspace/modules/xmlui/src/main/java/org/datadryad/dspace/xmlui/aspect/browse/ItemViewer.java

```diff
@@ -333,7 +333,10 @@ public void addPageMeta(PageMeta pageMeta) throws SAXException,
             // THIS IS A PACKAGE ITEM
 
             pageMeta.addMetadata("authors", "package").addContent(DryadWorkflowUtils.getAuthors(item));
-            pageMeta.addMetadata("metadata","json-ld").addContent(new DryadDataPackage(item).getSchemaDotOrgJSON());
+            String jsonData = new DryadDataPackage(item).getSchemaDotOrgJSON();
+            if (!jsonData.isEmpty()) {
+                pageMeta.addMetadata("metadata", "json-ld").addContent(new DryadDataPackage(item).getSchemaDotOrgJSON());
```

Comment:
Instead of `new DryadDataPackage(item).getSchemaDotOrgJSON()`  just use `jsonData`